src/components/CandleCharts.js

```diff
@@ -0,0 +1,83 @@
+import { VictoryChart,VictoryCandlestick,VictoryAxis,VictoryTheme,VictoryZoomContainer,VictoryScatter,VictoryCursorContainer } from "victory";
+import { useEffect,useState } from "react";
+const CandleCharts=()=>{
+
+
+  const axios = require('axios')
```

Comment:
Please use below code for importing axios
import axios from 'axios'

Reply:
Comment addressed.Please review